bentoml/frameworks/catboost.py

```diff
@@ -0,0 +1,118 @@
+import os
+
+from bentoml.exceptions import InvalidArgument, MissingDependencyException
+from bentoml.service.artifacts import BentoServiceArtifact
+from bentoml.service.env import BentoServiceEnv
+
+
+class CatBoostModelArtifact(BentoServiceArtifact):
+    """
+    Abstraction for saving/loading CatBoost models.
+
+    Args:
+        name (string): name of the artifact
+        model_type (string): Type of the model - 'classifier', 'regressor', 'catboost'
+        model_extension (string): Extension name for saved CatBoost model. Default - "cbm"
+        model_export_parameters (dict): Additional format-dependent parameters. Default - None
+        model_pool: The dataset previously used for training. See catboost.Pool for more details. Default - None.
```

Comment:
Would the users know ahead of time those two options before training the CatBoost model?

If these two options are changed with newly trained model, I think they would be better fit with the `pack` function.
```
svc.pack('model' CatBoost_Model, model_export=parameters={...}, model_pool={})
```